.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,7 @@
+-   id: flakehell
+    name: flakehell
+    description: '`flakehell` is `flake8` wrapper to make a nice, legacy-friendly, configurable command-line utility for enforcing style consistency across Python projects.'
```

Comment:
Grammar nitpick:

> `flakehell` is [**_a_**] `flake8 ` wrapper to make it nice, legacy-friendly, configurable ...

(without the square brackets, those are just there to make sure the single-letter word `a` stands out 😃 )